### PR TITLE
Slide build instructions added to README

### PR DESCRIPTION
**Pull request by paulodicanio86**
Added build instructions at the beginning, tested locally. Additionally some minor edits of the instructions throughout.

### CODE REVIEW

**Comment by paulodicanio86** [comment]
Added code to check the correct input and output path. Clarified the usage of 'screensaver/photos' in README.

File: screensaver/README.md [hunk 1]
```diff
@@ -3,15 +3,15 @@
 Given a list of directories containing photos of different trips or themes,
 this script allows you to randomly select one and upload to a location from which a 
 photo rendering app reads from, e.g. your laptop's screensaver application or 
-a similar one in a Raspberry Pi
+a similar one in a RPi
```
Comment (tsando) [inline screensaver/README.md]: Spelling `in a RPi` -> `in an RPi`.

File: screensaver/README.md [hunk 2]
```diff
@@ -3,15 +3,15 @@
...
 
 # Usage
 
 The script `screensaver.py` requires `python 3.7` with `numpy` and setting 3 environment variables:
 
-1. `SCREENSAVER_INPUT_PATH` (the path to where the photos directories are located)
-2. `SCREENSAVER_OUTPUT_PATH` (the path to where to upload the photos so the photo rendering app can read from)
-3. `SCREENSAVER_RSYNC_PORT` (the port over which ssh is being used, usually 22)
+1. `SCREENSAVER_INPUT_PATH` (The path to where the photos directories are located. Needs to end with `/`!)
+2. `SCREENSAVER_OUTPUT_PATH` (The path to where to upload the photos. Should not end with `/`)
```
Comment (tsando) [inline screensaver/README.md]: I see you somehow built these requirements into the code here https://github.com/tsando/photo-manager/blob/565be0916dca43db37b020f2e493e364c1c51630/screensaver/screensaver.py#L17-L21 so do we still need to ask users to how to end the input and output paths?

File: screensaver/screensaver.py [hunk 3]
```diff
@@ -14,6 +14,12 @@
 OUTPUT_PATH = os.getenv('SCREENSAVER_OUTPUT_PATH')
 RSYNC_PORT = os.getenv('SCREENSAVER_RSYNC_PORT')
 
+# Global variable path validation
+if INPUT_PATH[-1] != '/':
+    INPUT_PATH += '/'
+if OUTPUT_PATH[-1] == '/':
+    OUTPUT_PATH = OUTPUT_PATH[:-1]
+
```
Comment (tsando) [inline screensaver/screensaver.py:22]: We should create a test for this in the future :)

**Review by tsando — COMMENTED** [review]
Thanks for improving the documentation and for fixing the input and output paths! They are very good! Please see comments before I can approve.

File: screensaver/screensaver.py (hunk 3 above)
Comment (paulodicanio86) [reply screensaver/screensaver.py:22]: Agree, for the future.

File: screensaver/README.md (hunk 2 above)
Comment (paulodicanio86) [reply screensaver/README.md]: Yes, I would say so, in case they miss it. But I will adjust the language.

File: screensaver/README.md (hunk 1 above)
Comment (paulodicanio86) [reply screensaver/README.md]: Agreed. will change.

**Review by tsando — APPROVED** [review]
LGTM!

**Pull request merged**